src/main/manpages/fish/payara/extras/diagnostics/asadmin/collect-diagnostics.1

```diff
@@ -29,8 +29,8 @@ OPTIONS
        --help, -?
            Displays the help text for the subcommand.
 
-       --serverlog
-            Whether to collect server logs. The default is true.
+       --logs
+            Whether to collect all logs. The default is true.
```

Comment:
Might be better to specify what logs instead of all (notification, server and access)